seguridad/views.py

```
@@ -25,6 +25,25 @@ def post(self, request, *args, **kwargs):
             data['error'] = str(e)
         return JsonResponse(data, status=200)
 
+    def put(self, request, *args, **kwargs):
+        data = {'resp': False}
+        try:
+            cuenta = str(request.POST).strip()
```

Ruff (0.11.9)

Local variable `cuenta` is assigned to but never used

Remove assignment to unused variable `cuenta`

(F841)